mozilla/activity-stream
# Preparing the release of Activity Stream 1.1.9

Hi all,

We are now preparing the release of A-S 1.1.9. The add-on has been made available at [pre-release channel](https://moz-activity-streams-prerelease.s3.amazonaws.com/dist/latest.html). Please install it, test it, and leave the feedback, bug reports, and other inputs in this thread.

# Highlights (Changelog)
* [new] Graduated the local metadata parsing 💯 
* [fix] Optimized the webpack build to reduce the add-on size by 25% 💯 
* [fix] Various bugfixes in TabTracker, ContextMenu, and ExperimentProvider
* [fix] Cleaned up the exception reporting
* [deprecated] The forked version of simple storage
* [deprecated] The recommended related code

# Release checklist
- QA
- Product
- Engineering

Please use the `release/block` or `release/uplift` to label the bug reports as well as other release related issues. Once you finish the review, please check off the corresponding item in the above checklist.

*Note*: checking off the item may not work, you can either modify it in place (i.e modify `[ ]` to `[x]`), or just drop a good old `r+` comment below.

[Edit]: I've updated the version of this release to 1.1.9, since we have had a chemspill release that, unfortunately, has taken the 1.1.8. My apologies for the confusion, and we are going to fix it to avoid any further version collisions.

Thanks


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/mozilla/activity-stream/1827)
